src/widget/woverview.cpp

```diff
@@ -955,13 +955,6 @@ void WOverview::drawMarkLabels(QPainter* pPainter, const float offset, const flo
                 continue;
             }
         }
-        if (pMark->m_label.intersects(m_cuePositionLabel)
-                || pMark->m_label.intersects(m_cueTimeDistanceLabel)
-                || pMark->m_label.intersects(m_timeRulerPositionLabel)
-                || pMark->m_label.intersects(m_timeRulerDistanceLabel)) {
-            continue;
-        }
```

Comment:
Revert this. Labels should not overlap.

Reply:
ping

Reply:
It is not the right place to force this here. This code skips to draw the label just because the font is too big or the waveform too small. I agree that the labels should not overlap, but if there is a skin issue it should be rendered anyway to see the overlap.

Reply:
This is required for normal functionality. It is not to work around bugs in skins. It is normal for the intro/outro cue markers and durations which are rendered on top, and the main cue label which is rendered at vertical center in Tango, to overlap the labels when holding right click.

> if there is a skin issue it should be rendered anyway to see the overlap.

I disagree. The label disappearing makes bugs easier to catch.

Reply:
Some pixel vertical overlap is no issue, rectifies not drawing the lable, especially because the position and distance labels are still painted overlapping. We need here a solution that detects the cue type and hides the labels based on that.

Reply:
No, we need to not have overlap. Please do not make this any more complicated. I will not merge this without this commit being reverted (or removed with a rebase).

Reply:
We need a solution for the root issue first: Different font heights on different systems. Will you fix it in your branch? 
Than I can finally adjust this code here.